snuba/consumers/consumer_builder.py

```diff
@@ -131,15 +133,14 @@ def __init__(
             commit_log_topic_spec = stream_loader.get_commit_log_topic_spec()
             if commit_log_topic_spec is not None:
                 self.commit_log_topic = Topic(commit_log_topic_spec.topic_name)
+                self.commit_log_producer = Producer(
+                    build_kafka_producer_configuration(commit_log_topic_spec.topic)
```

Comment:
How come you didn't pass the bootstrap servers here?

Reply:
Oops, good catch